src/main/java/Duke.java

```diff
@@ -1,11 +1,22 @@
 import java.util.Scanner;
 
 public class Duke{
+    public static final int MAX_SIZE = 100;
+
+    public static void main(String[] args) {
+        Task[] tasks = new Task[MAX_SIZE];
+        for (int i=0; i<MAX_SIZE; i++) {
```

Comment:
Perhaps you could leave some space in the loop forming stage, for example i < MAX_SIZE; to improve readability.

Reply:
Will look into that in my subsequent commits. Thank you for your comments!